frontend/app/views/fragments/tier/packagePromoSupporter.scala.html

```diff
@@ -31,6 +31,7 @@ <h4 class="u-margin-top">Benefits:</h4>
             <ul class="o-bulleted-list">
                 <li>Welcome pack, card and gift</li>
                 <li>Regular Members updates</li>
+                <li>Enjoy an ad-free experience with free access to the premium tier of the app</li>
```

Comment:
Consider adding this to Benefits.scala maybe
and perhaps powering this view off it